### ISSUE TITLE
Death Message 

### ISSUE THREAD

**Issue by MateoxPL** [FunnyGuilds/FunnyGuilds#1046]
Witam
Chciałbym zgłosić problem z wyświetlaniem Death Message FG, gdy wyłączymy powiadomienia o śmierci z głównego MC poprzez gamerule lub jakiś plugin to również wiadomości o zabójstwie nie jest wyświetlana (zgłaszałem ten problem na discordzie lecz nie znalazłem odpowiedniego rozwiązania) moim zdanie wystarczyło by dodać zwykły broadcast wysyłający do wszystkich graczy
Pozdrawiam

**Comment by Kamilkime** [comment]
No... Jak dla mnie to może być najwyżej propozycja zmiany systemu, a nie problem - to wiadomość o śmierci, więc jest dokładnie tak robiona obecnie, moim zdaniem tak właśnie powinno być

**Comment by MateoxPL** [comment]
Dokładnie może być opcja w configu np Ignore-DisabledDeathMessages:
